resources/views/kiosk.blade.php

```diff
@@ -1,5 +1,108 @@
+                            <th scope="col">User</th>
+                            <th>Created At</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                            @foreach($activities as $activity)
+                            <tr>
+                                <th scope="row">{{ $activity->log_name }}</th>
+                                <td>{{ $activity->description }}</td>
+                                <td>
+                                    {{ $activity->causer->name ?? __('Unknown user') }}
+                                </td>
+                                <td>
+                                    {{ $activity->created_at->format('d/m/Y - H:i:s') }}
+                                </td>
+                            </tr>
+                            @endforeach
+                            <tr class="text-center">
+                                <td colspan="4">
+                                    <a href="{{ kioskRoute('activity.index') }}">See more</a>
+                                </td>
+                            </tr>
+                        </tbody>
+                      </table>
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="text-center">
+                    <h3>Latest Created User</h3>
+                </div>
+                <div class="table-responsive">
+                    <table class="table table-sm mb-0 table-hover">
+                        <thead class="text-center">
+                          <tr>
+                            <th scope="col">User Group</th>
+                            <th scope="col">E-Mail</th>
+                            <th>Name</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                            @foreach($users as $user)
+                            <tr>
+                                <td>{{ $user->user_group }}</td>
+                                <td>{{ $user->email }}</td>
+                                <td>{{ $user->name }}</td>
+                            </tr>
+                            @endforeach
+                            <tr class="text-center">
+                                <td colspan="3">
+                                    <a href="{{ kioskRoute('users.index') }}">See more</a>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="text-center">
+                    <h3>Latest Created Roles</h3>
+                </div>
+                <div class="table-repsonsive">
+                    <table class="table table-sm mb-0 table-hover">
+                        <thead class="text-center">
+                          <tr>
+                            <th scope="col">Role Name</th>
```

Comment:
Convent the table headings plz from `Role name` to `__('Role name')` by example

Reply:
Hi, currently I just push the fixing